includes/Admin/Orders.php

```diff
@@ -103,7 +103,45 @@ public function enqueue_assets() {
 	 * @since 2.1.0-dev.1
 	 */
 	public function add_notices() {
+		global $post;
+
+		if ( ! $this->is_edit_order_screen() ) {
+			return;
+		}
+
+		$order  = wc_get_order( $post );
+		$plugin = facebook_for_woocommerce();
+
+		if ( Commerce\Orders::is_order_pending( $order ) ) {
 
+			/* translators: Placeholders: %1$s - <strong> tag, %2$s - </strong>, %3$s - <strong> tag, %4$s - </strong> tag */
+			$message = sprintf(
+				__( 'This order is currently being held by Instagram and cannot be edited. Once released by Instagram, it will move to %1$sProcessing%2$s or %3$sCancelled%4$s status.', 'facebook-for-woocommerce' ),
+				'<strong>', '</strong>',
+				'<strong>', '</strong>'
+			);
+
+			$plugin->get_admin_notice_handler()->add_admin_notice( $message, $plugin::PLUGIN_ID . '_commerce_order_pending_' . $order->get_id(), [
+				'dismissible'  => true,
+				'notice_class' => 'notice-info',
+			] );
+		}
+
+		if ( get_transient( 'wc_facebook_bulk_order_update' ) ) {
+
+			/* translators: Placeholders: %1$s - <strong> tag, %2$s - </strong> */
+			$message = sprintf(
+				__( '%1$sHeads up!%2$s Instagram order statuses can’t be updated in bulk. Please update Instagram orders individually so you can provide order details required by Instagram.', 'facebook-for-woocommerce' ),
+				'<strong>', '</strong>'
+			);
+
+			$plugin->get_admin_notice_handler()->add_admin_notice( $message, $plugin::PLUGIN_ID . '_facebook_bulk_order_update', [
```

Comment:
I'd probably remove the second "facebook" or replace it with "commerce".

Reply:
_Related to my most recent comment._

This code is now part of another PR that was already merged. Since this is a minor change, maybe we can change this ID while working on another `Admin\Order` story.